python/ray/experimental/state/api.py

```diff
@@ -119,8 +118,9 @@ def __init__(
         """Initialize a StateApiClient and check the connection to the cluster.
 
         Args:
-            address: The address of Ray API server. If not provided,
-                it will be configured automatically from querying the GCS server.
+            address: The URL of Ray API server. E.g. `http://localhost:8265`.
```

Comment:
As discussed, this should be the GCS address (and I believe that's the implementation). Can you double check it?